CONTRIBUTING.md

```diff
@@ -11,14 +12,22 @@ If you want to learn more about this project, read the [wiki](https://github.com
 
 ## Contributions
 
+Please review this [diagram](https://drive.google.com/file/d/1RjCeUwJ0s3huq1iRSXwdy02wD54HKNxA/view?usp=sharing), to better understand the git workflow.
+
 We would be pleased to accept code contributions to this project.
-As we are in the alpha stage, the main priority is getting a robust blockchain up and running, with as many of the mainnet features as possible.
+As we have now released, the main priority is improving the mainnet blockchain.
 You can visit our [Trello project board](https://trello.com/b/ZuNx7sET) to get a sense of what is in the backlog.
 Generally things to the left are in progress or done. Some things go through "Coming up soon" but some will come directly out of other columns.
 Usually the things closer to the top of each column are the ones that will be worked on soonest.
 If you are interested in cryptography, math, or just like hacking in python, there are many interesting problems to work on.
 Contact any of the team members on [Keybase](https://keybase.io/team/chia_network.public), which we use as the main communication method and you can comment on any Trello card.
 
+We ask that external contributors create a fork of the `main` branch for any feature work they wish to take on.
+
+Members of the Chia organization may create feature branches from the `main` branch.
+
+In the event an emergency fix is required for the release version of Chia, members of the chia organization will create a feature branch from the current release branch `1.0.0`
```

Comment:
May not want to say `1.0.0` because that means when we move on to 1.2 & 1.1, we have to update this file (which we will almost certainly forget to do).

Reply:
All actual released versions have to be three dots. Aka 1.1 is actually 1.1.0 for semver reasons

Reply:
I want to make this document j2 templated and make it a release artifact. 😃